src/Tests/Microsoft.NET.Clean.Tests/GivenThatWeWantToCleanAProject.cs

```diff
@@ -53,5 +53,39 @@ public void It_cleans_without_logging_assets_message()
                 .And
                 .NotHaveStdOutContaining("warning");
         }
+
+        [Fact]
+        public void It_cleans_without_assets_file_present()
+        {
+            var testAsset = _testAssetsManager
+                .CopyTestAsset("HelloWorld")
+                .WithSource();
+
+            var lockFilePath = Path.Combine(testAsset.TestRoot, "obj", "project.assets.json");
+            File.Exists(lockFilePath).Should().BeFalse();
+
+            var cleanCommand = new CleanCommand(Log, testAsset.TestRoot);
+
+            cleanCommand
+                .Execute()
+                .Should()
+                .Pass();
+        }
+
+        [Fact]
+        public void It_can_clean_and_build_without_using_rebuild()
+        {
+            var testAsset = _testAssetsManager
+              .CopyTestAsset("HelloWorld")
+              .WithSource()
+              .Restore(Log);
+
+            var cleanAndBuildCommand = new CleanCommand(Log, testAsset.TestRoot);
+
+            cleanAndBuildCommand
+                .Execute("/t:Build")
```

Comment:
I think it would be clearer to use an `MSBuildCommand` here and pass in `/t:Clean;Build` as targets.